docs/CHANGELOG.md

```diff
@@ -9,8 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Racetrack client keeps the local config file `~/.racetrack/config.yaml` with mode `600`
   (not readable by others) and warns you if this file has insecure permissions
   since it may contain credentials.
+- Endpoint `/api/v1/auth/token/user/regenerate`'s previous functionality moved to `/api/v1/auth/token/user/all/regenerate 
 ### Added
-- Endpoint `/api/v1/auth/token/user/<username>/regenerate` added to regenerate tokens on a per-user basis.
+- Endpoint `/api/v1/auth/token/user/regenerate` added to regenerate tokens on a per-user basis.
```

Comment:
Nitpicking: In my opinion there's no need to be so specific. Notice that we just want to keep **user-facing** changes in here. So probably "There is a new button to regenerate your auth token" will be interesting enough for our "users".